sampleconfig/configtx.yaml

```diff
@@ -141,19 +141,22 @@ Application: &ApplicationDefaults
         # (e.g.,who can receive Block events). This section does NOT specify the resource's
         # definition or API, but just the ACL policy for it.
         #
-        # User's can override these defaults with their own policy mapping by defining the
+        # Users can override these defaults with their own policy mapping by defining the
         # mapping under ACLs in their channel definition
 
         #---New Lifecycle System Chaincode (_lifecycle) function to policy mapping for access control--#
 
+        # ACL policy for _lifecycle's "CheckCommitReadiness" function
+        _lifecycle/CheckCommitReadiness: /Channel/Application/Writers
```

Comment:
Seems like this one should be /Channel/Application/Readers ?

Reply:
I was on the fence here. I think CheckCommitReadiness is mostly intended to be run by an admin right before they commit the chaincode definition. It could be confusing (or at least less than user-friendly) if a user is able to check the commit readiness successfully but then is blocked from performing the actual commit since they're a channel reader but not a writer.

Reply:
https://jira.hyperledger.org/browse/FAB-17412?focusedCommentId=66842&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-66842